teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative('./person')
+
+class Teacher < Person
+  # def initialize(*args, specialization, **kwargs)
+  #   super(*args, **kwargs)
+  #   @specialization = specialization
+  # end
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
all errors has been corrected